[2023-07-11T14:46:41.145+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: real_estate_etl_dag.transform_task manual__2023-07-11T14:45:57.353441+00:00 [queued]>
[2023-07-11T14:46:41.160+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: real_estate_etl_dag.transform_task manual__2023-07-11T14:45:57.353441+00:00 [queued]>
[2023-07-11T14:46:41.160+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-07-11T14:46:41.160+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 4
[2023-07-11T14:46:41.160+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-07-11T14:46:41.175+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): transform_task> on 2023-07-11 14:45:57.353441+00:00
[2023-07-11T14:46:41.183+0000] {standard_task_runner.py:55} INFO - Started process 41880 to run task
[2023-07-11T14:46:41.187+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'real_estate_etl_dag', 'transform_task', 'manual__2023-07-11T14:45:57.353441+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/real_estate_etl_dag.py', '--cfg-path', '/tmp/tmpnotf12vj']
[2023-07-11T14:46:41.190+0000] {standard_task_runner.py:83} INFO - Job 87: Subtask transform_task
[2023-07-11T14:46:41.314+0000] {task_command.py:389} INFO - Running <TaskInstance: real_estate_etl_dag.transform_task manual__2023-07-11T14:45:57.353441+00:00 [running]> on host ip-172-31-34-219.ap-southeast-2.compute.internal
[2023-07-11T14:46:41.449+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=real_estate_etl_dag
AIRFLOW_CTX_TASK_ID=transform_task
AIRFLOW_CTX_EXECUTION_DATE=2023-07-11T14:45:57.353441+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-07-11T14:45:57.353441+00:00
[2023-07-11T14:46:41.483+0000] {logging_mixin.py:137} INFO - DATAFRAME DIMENSIONS: (0, 0)
[2023-07-11T14:46:41.484+0000] {logging_mixin.py:137} INFO - None
[2023-07-11T14:46:41.484+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ubuntu/venv/lib/python3.10/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/ubuntu/venv/lib/python3.10/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/ubuntu/airflow/dags/real_estate_etl_dag.py", line 76, in clean_data
    df = df[df['price_per_week'].str.contains('\$') & df['price_per_week'].str.contains('\d')]
  File "/home/ubuntu/venv/lib/python3.10/site-packages/pandas/core/frame.py", line 3761, in __getitem__
    indexer = self.columns.get_loc(key)
  File "/home/ubuntu/venv/lib/python3.10/site-packages/pandas/core/indexes/range.py", line 349, in get_loc
    raise KeyError(key)
KeyError: 'price_per_week'
[2023-07-11T14:46:41.509+0000] {taskinstance.py:1322} INFO - Marking task as UP_FOR_RETRY. dag_id=real_estate_etl_dag, task_id=transform_task, execution_date=20230711T144557, start_date=20230711T144641, end_date=20230711T144641
[2023-07-11T14:46:41.519+0000] {standard_task_runner.py:100} ERROR - Failed to execute job 87 for task transform_task ('price_per_week'; 41880)
[2023-07-11T14:46:41.562+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-07-11T14:46:41.599+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
